src/main/python/systemml/defmatrix.py

```diff
@@ -413,7 +417,7 @@ class matrix(object):
        Then the left-indexed matrix is set to be backed by DMLOp consisting of following pydml:
        left-indexed-matrix = new-deep-copied-matrix
        left-indexed-matrix[index] = value
-    8. Please use m.print_ast() and/or  type `m` for debugging. Here is a sample session:
+    8. Please use m._print_ast() and/or  type `m` for debugging. Here is a sample session:
```

Comment:
The function name change for this internal function looks good, but I think the documentation is wrong now -- the user should still use the matrix `print_ast()` function, right?